### ISSUE TITLE
[Vertical Taskbar for Windows 11] Background for the taskbar is off by one pixel

### ISSUE THREAD

**Issue by Venthe** [ramensoftware/windhawk-mods#1017]
There is a single pixel of the background not aligning which is present on the right; regardless of the orientation of the vertical taaskbar.

* Right: ![image](https://github.com/user-attachments/assets/9d084bd6-b43b-4a09-8a9b-a628abb6d98c)
* Left: ![image](https://github.com/user-attachments/assets/08d63fd1-ddc2-41e1-98af-f8739d36f058)

I'm almost sure that this is a regression in `1.2.3`. I've also disabled all the other mods; tested other taskbar widths and killed `explorer` and `dwm`

```json
{"TaskbarWidth":48,"taskbarLocation":"right","taskbarLocationSecondary":"sameAsPrimary","startMenuWidth":0}
```

**Comment by SandTechStuff** [comment]
I noticed this too, it’s really easy to see if you maximize a program.

**Comment by m417z** [comment]
Fixed in version 1.2.4.

**Comment by Venthe** [comment]
Confirmed working, thanks
